### ISSUE TITLE
can I use it in nuxt3?

### ISSUE THREAD

**Issue by youkei-zzz** [zernonia/vue3-notion#44]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by dasturchioka** [comment]
can I use it in Nuxt3?

**Comment by hacknug** [comment]
> can I use it in Nuxt3?

Of course! Check the readme for instruction on how to install and use the module in Nuxt: https://github.com/zernonia/vue3-notion?tab=readme-ov-file#nuxt3-module
